README.md

```diff
@@ -1 +1,69 @@
 # React-boilerplate
+
+## link for my digital business card: https://nngrz.github.io/business-card
+
+## What I have done:
+I built a React app from scratch called "business-card" that showcases my personal information. The app consists of components for sections like Info, About, Interests, and Footer. Additionally, for practice, I added an extra section called Portfolio, where users can visit one of my projects via a link. I successfully deployed the app to GitHub Pages for public access.
```

Comment:
Please make it look nicer - you can preview markdown in thi PR.

Please:
1. Don't use "I": "I build", "I added". Instead say in a declarative mode, saying what this repo consists of. For example: "Digital Portfolio of Ningrui. Consists of all of my projects that I have worked on with it's source code "